storm-client/src/jvm/org/apache/storm/task/WorkerTopologyContext.java

```diff
@@ -34,6 +36,8 @@
     private String _pidDir;
     Map<String, Object> _userResources;
     Map<String, Object> _defaultResources;
+    private AtomicReference<Map<Integer, NodeInfo>> _taskToNodePort;
+    private String _assignmentId;
```

Comment:
Could we avoid adding in new member variables with _ as the first character.  This is against the new guidelines.